### PR TITLE
CEDS-1405 - model change for containers and seals

### PR DESCRIPTION
**Pull request by SteveSugden**
This is the back-end model change for containers and seals.   It will probably break existing acceptance tests (although, declarations with containers/seals are already broken to some extent) so I'd advise not merging until the frontend changes are ready.

### CODE REVIEW

File: app/uk/gov/hmrc/exports/services/mapping/goodsshipment/consignment/TransportEquipmentBuilder.scala [hunk 1]
```diff
@@ -17,24 +17,33 @@
...
+    extends ModifyingBuilder[TransportInformationContainers, GoodsShipment.Consignment] {
+  override def buildThenAdd(containersHolder: TransportInformationContainers, consignment: Consignment): Unit =
+    consignment.getTransportEquipment.addAll(
+      containersHolder.containers.zipWithIndex.map(data => createTransportEquipment(data)).toList.asJava
+    )
 
-  private def createTransportEquipment(seals: Seq[Seal]): Seq[GoodsShipment.Consignment.TransportEquipment] = {
+  private def createTransportEquipment(
+    containerData: (TransportInformationContainer, Int)
+  ): GoodsShipment.Consignment.TransportEquipment = {
```
Comment (PatrykRudnicki) [inline app/uk/gov/hmrc/exports/services/mapping/goodsshipment/consignment/TransportEquipmentBuilder.scala]: Could you import `GoodsShipment.Consignment.TransportEquipment`?

File: test/util/testdata/ExportsTestData.scala [hunk 2]
```diff
@@ -92,7 +92,12 @@ object ExportsTestData {
         <ns3:GoodsShipment>
           <ns3:Consignment>
             <ns3:TransportEquipment>
-              <ns3:SequenceNumeric>0</ns3:SequenceNumeric>
+              <ns3:SequenceNumeric>1</ns3:SequenceNumeric>
+              <ns3:ID>container123</ns3:ID>
+              <ns3:Seal>
+                <ns3:SequenceNumeric>1</ns3:SequenceNumeric>
```
Comment (bambuchaAdm) [inline test/util/testdata/ExportsTestData.scala]: very minor - formats

**Comment by bambuchaAdm** [comment]
I assume this is just backend part and we assume that all seals in declaration are applied to all containers ?

**Comment by SteveSugden** [comment]
No.  A container can contain multiple seals.   So seals are part of a container, not the declaration.

**Pull request merged**